Retriggers of master are failing on Travis

I have a branch locally for adding Django 1.10/1.11 to the testing matrix.

However I noticed that even the current testing matrix is currently failing, on fresh pushes of master to my fork. eg:

1) https://travis-ci.org/edmorley/hawkrest/jobs/193424762
> ImportError: cannot import name setup

(presumably newer django-nose is not compatible with Django 1.6):

2) https://travis-ci.org/edmorley/hawkrest/jobs/193424766
> AssertionError: Unexpected call: log.warning("access denied: MisComputedContentHash: Our hash blPLW7mrURX5LO7m2f7iW+3kXihtvlX0//IigunDeCs= (sha256) did not match theirs e8gjS2dr/QZv4GEJa8HHIsupVVrGiWA7YVZnVStNv3c=")

We should first pin these dependencies to make master green, before I make further changes to the testing matrix.

The last (green) Travis run on master was:
https://travis-ci.org/kumar303/hawkrest

Comparing the 'py27-django1.8-drf3.3' job to that from my run, shows the following package changes:
```diff
--- working.txt 2017-01-19 17:07:00.306042800 +0000
+++ broken.txt  2017-01-19 17:06:46.381880000 +0000
@@ -1,26 +1,28 @@
-alabaster==0.7.8
+alabaster==0.7.9
+args==0.1.0
 Babel==2.3.4
-Django==1.8.13
-django-nose==1.4.3
+clint==0.5.1
+Django==1.8.17
+django-nose==1.4.4
 djangorestframework==3.3.3
-docutils==0.12
+docutils==0.13.1
 funcsigs==1.0.2
 hawkrest==0.0.10
 imagesize==0.7.1
-Jinja2==2.8
+Jinja2==2.9.4
 MarkupSafe==0.23
 mock==2.0.0
-mohawk==0.3.2.1
+mohawk==0.3.4
 nose==1.3.7
 nosenicedots==0.5
 pbr==1.10.0
-pkginfo==1.3.2
+pkginfo==1.4.1
 Pygments==2.1.3
-pytz==2016.4
-requests==2.10.0
-requests-toolbelt==0.6.2
+pytz==2016.10
+requests==2.12.5
+requests-toolbelt==0.7.0
 six==1.10.0
 snowballstemmer==1.2.1
-Sphinx==1.4.2
+Sphinx==1.5.1
 sphinx-rtd-theme==0.1.9
-twine==1.6.5
+twine==1.8.1
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -10,9 +10,6 @@ python: "3.5"
 env:
     - TOX_ENV=docs
 
-    - TOX_ENV=py27-django1.6-drf3.2
-    - TOX_ENV=py27-django1.7-drf3.2
-    - TOX_ENV=py27-django1.7-drf3.3
     - TOX_ENV=py27-django1.8-drf3.2
     - TOX_ENV=py27-django1.8-drf3.3
     - TOX_ENV=py27-django1.9-drf3.3

--- a/docs/index.rst
+++ b/docs/index.rst
@@ -51,7 +51,7 @@ Installation
 Requirements:
 
 * Python 2.7+ or 3.4+
-* `Django`_ 1.6 through 1.9
+* `Django`_ 1.8 through 1.9
 * `Django Rest Framework`_ 3.2 or 3.3
 * `mohawk`_
 (Older versions of these libraries may work, but support is not guaranteed.)
@@ -92,6 +92,10 @@ Changelog
     If you're upgrading from a version prior to 0.0.6, be sure to
     use ``rest_framework.permissions.IsAuthenticated`` on your views
     :ref:`as documented <protecting-api-views>`
+
+- **Unreleased**
+
+  - Dropped support for Django 1.6/1.7.
 
 - **0.0.10** (2016-06-01)
 

--- a/setup.py
+++ b/setup.py
@@ -22,4 +22,4 @@ setup(name='hawkrest',
           'Topic :: Internet :: WWW/HTTP',
       ],
       packages=find_packages(exclude=['tests']),
-      install_requires=['djangorestframework', 'mohawk>=0.3.0'])
+      install_requires=['djangorestframework', 'mohawk>=0.3.3'])

--- a/tests/test_authentication.py
+++ b/tests/test_authentication.py
@@ -45,7 +45,7 @@ class AuthTest(BaseTest):
     def assert_log_regex(self, method, pattern):
         log_call = getattr(self.mock_log, method).call_args[0][0]
         assert re.search(pattern, log_call), (
-            'Unexpected call: log.{}("{}")'.format(method, log_call))
+            'Expected log.{}() matching "{}", saw: "{}"'.format(method, pattern, log_call))
 
 
 class TestAuthentication(AuthTest):

--- a/tests/test_authentication.py
+++ b/tests/test_authentication.py
@@ -106,7 +106,7 @@ class TestAuthentication(AuthTest):
         self.assertRaisesRegexp(AuthenticationFailed,
                                 '^Hawk authentication failed$',
                                 lambda: self.auth.authenticate(req))
-        self.assert_log_regex('warning', '^access denied: MacMismatch: ')
+        self.assert_log_regex('warning', '^access denied: MisComputedContentHash: ')
 
     def test_hawk_get_wrong_sig(self):
         sender = self._sender(url='http://realsite.com')

--- a/tests/test_middleware.py
+++ b/tests/test_middleware.py
@@ -2,7 +2,7 @@ from django.conf import settings
 from django.http import HttpResponse
 
 import mock
-from mohawk.exc import MacMismatch
+from mohawk.exc import MisComputedContentHash
 from mohawk import Receiver
 
 from hawkrest import lookup_credentials
@@ -57,7 +57,7 @@ class TestMiddleware(BaseTest):
 
         response.content = 'TAMPERED WITH'
 
-        with self.assertRaises(MacMismatch):
+        with self.assertRaises(MisComputedContentHash):
             self.accept_response(res, sender)
 
     def test_respond_with_bad_content_type(self):
@@ -68,5 +68,5 @@ class TestMiddleware(BaseTest):
 
         response['Content-Type'] = 'TAMPERED WITH'
 
-        with self.assertRaises(MacMismatch):
+        with self.assertRaises(MisComputedContentHash):
             self.accept_response(res, sender)

--- a/tox.ini
+++ b/tox.ini
@@ -4,8 +4,6 @@
 # When updating the envlist, be sure to also update TOX_ENV in .travis.yml
 envlist =
     docs,
-    py27-django1.6-drf3.2,
-    py27-django1.7-drf{3.2,3.3},
     {py27,py34,py35}-django1.8-drf{3.2,3.3}
     # Django 1.9 requires DRF >= 3.3
     {py27,py34,py35}-django1.9-drf3.3
@@ -13,8 +11,6 @@ envlist =
 [testenv]
 commands = {envpython} runtests.py []
 deps =
-    django1.6: Django>=1.6,<1.7
-    django1.7: Django>=1.7,<1.8
     django1.8: Django>=1.8,<1.9
     django1.9: Django>=1.9,<1.10
     drf3.2: djangorestframework>=3.2,<3.3

